ghost/i18n/locales/nl/portal.json

```diff
@@ -1,148 +1,149 @@
+  "Emails disabled": "E-mails zijn uitgeschakeld",
+  "Ends {{offerEndDate}}": "Eindigt op {{offerEndDate}}",
+  "Error": "Fout",
+  "Expires {{expiryDate}}": "Verloopt op {{expiryDate}}",
+  "Forever": "Voor altijd",
+  "Free Trial – Ends {{trialEnd}}": "Gratis proefperiode - Eindigt {{trialEnd}}",
+  "Get help": "Lees meer",
+  "Get in touch for help": "Neem contact op voor hulp",
+  "Get notified when someone replies to your comment": "Blijf op de hoogte als iemand op jouw reactie reageert",
+  "Give feedback on this post": "Deel je mening over dit artikel",
+  "Help! I'm not receiving emails": "Help! Ik ontvang geen e-mail",
+  "If a newsletter is flagged as spam, emails are automatically disabled for that address to make sure you no longer receive any unwanted messages.": "Als een nieuwsbrief als spam wordt gemarkeerd, worden e-mails automatisch uitgeschakeld voor dat adres om ervoor te zorgen dat je geen ongewenste berichten meer ontvangt.",
+  "If the spam complaint was accidental, or you would like to begin receiving emails again, you can resubscribe to emails by clicking the button on the previous screen.": "Als de spamklacht per ongeluk was, of als je weer e-mails wil ontvangen, kun je zich opnieuw abonneren op e-mails door op de knop op het vorige scherm te klikken.",
+  "If you cancel your subscription now, you will continue to have access until {{periodEnd}}.": "Als je je abonnement nu opzegt, heb je toegang tot {{periodEnd}}.",
+  "If you have a corporate or government email account, reach out to your IT department and ask them to allow emails to be received from {{senderEmail}}": "Als je een zakelijk of overheids e-mailaccount heeft, neem je contact op met uw IT-afdeling en vraag je hen toestemming te geven om e-mails te ontvangen van {{senderEmail}}",
+  "If you would like to start receiving emails again, the best next steps are to check your email address on file for any issues and then click resubscribe on the previous screen.": "Als je weer e-mails wilt ontvangen, kun je het beste eerst uw e-mailadres in ons bestand controleren op eventuele problemen en vervolgens op opnieuw abonneren klikken in het vorige scherm.",
+  "If you're not receiving the email newsletter you've subscribed to, here are few things to check.": "Als je de e-mailnieuwsbrief waarop je je hebt geabonneerd niet ontvangt, moet je het volgende controleren.",
+  "If you've completed all these checks and you're still not receiving emails, you can reach out to get support by contacting {{supportAddress}}.": "Als je al deze controles hebt voltooid en je nog steeds geen e-mails ontvangt, kun je contact opnemen voor ondersteuning door contact op te nemen met {{supportAddress}}.",
+  "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "In het geval dat er een permanente fout wordt ontvangen bij het verzenden van een nieuwsbrief, worden e-mails uitgeschakeld op het account.",
```

Comment:
In het geval dat er een permanente fout optreedt bij het verzenden van een nieuwsbrief, worden e-mails uitgeschakeld op dat account.